src/Umbraco.Cms.Integrations.SEO.GoogleSearchConsole.UrlInspectionTool/App_Plugins/UmbracoCms.Integrations/SEO/GoogleSearchConsole/URLInspectionTool/urlInspection.html

```diff
@@ -0,0 +1,125 @@
+﻿<div ng-controller="UmbracoCms.Integrations.GoogleSearchConsole.UrlInspectionToolController as vm">
+    <umb-load-indicator ng-if="vm.loading"></umb-load-indicator>
+    <umb-box>
+        <umb-box-header title-key="urlInspectionTool_title">
+            <umb-button action="vm.onConnectClick()"
+                        type="button"
+                        button-style="primary"
+                        state="init"
+                        label="Connect"
+                        disabled="vm.oauthConfiguration.isConnected">
+            </umb-button>
+            <umb-button action="vm.onRevokeToken()"
+                        type="button"
+                        button-style="danger"
+                        state="init"
+                        label="Revoke"
+                        disabled="!vm.oauthConfiguration.isConnected">
+            </umb-button>
+        </umb-box-header>
+        <umb-box-content>
+            <div>
+                <h5>About Google Search Console - URL Inspection API</h5>
+                <p>
+                    The Search Console APIs are a way to access data outside of Search Console, through external applications and products.
```

Comment:
I'd suggest look to reduce this to just a couple of paragraphs and see if you can make it a little more editor friendly.  I.e. talking less about APIs, parameters and responses and more about relevant information about the page that is used by Google to determine the search engine ranking.

Do keep the bit about usage limits though, that's important for them to know.